### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by yogi-lo** [vitorpamplona/amethyst#568]
**Describe the bug**
Parsing error --cannot send zaps. (However, I am able to receive zaps)

**To Reproduce**
1. Go to note. 
2. Click on zap. 
3. See error message attached. 
![Screenshot_20230904-125604~2](https://github.com/vitorpamplona/amethyst/assets/53883084/319b4ed0-9f38-4a63-acbe-3adb1f4206d8)

I have upgraded to ver . 75.14

**Expected behavior**
I expected to send sats. I tested with multiple users and zapping function is failing. 

**Video and Screenshots**
See attached error message. 

**Device (please complete the following information):**
 - Phone Brand/Model [e.g. Pixel 7 Pro]: pixel 6
 - Android Version [e.g. 33]: 13
 - App Version [e.g. v0.20.3]: .75.14

**Bounty (in Bitcoin sats) offered for a solution**
100 sats when it's fixed. Will send via amethyst as I don't know how else to do send. Thank you ✨🙏

**Comment by vitorpamplona** [comment]
Does this happen with every receiver?

**Comment by yogi-lo** [comment]
There was one receiver who said she received my zap, in spite of the error message.

Everyone else said they did not receive.

I just logged into the alby chrome extension and under outgoing transactions for this account it only has one listed

14 days ago to a paid relay I added.
I wasn't sure what I was doing when I clicked the link and was surprised that I had to set up an alby extension, bc I thought I already had an alby extension (with my anon account that I previously set up)... But perhaps I was only using alby browser

I recall getting some error messages at that time, but I was receiving sats so I was not concerned.

So perhaps this is related to that relay?

Sent with [Proton Mail](https://proton.me/) secure email.

------- Original Message -------
